src/main/app/fees/feesClient.ts

```diff
@@ -23,8 +24,8 @@ export class FeesClient {
    * @returns {Promise.<number>} promise containing the fee amount in pounds
    */
   static async calculateIssueFee (claimValue: number): Promise<number> {
-    const fee: CalculationOutcome = await this.calculateFee(issueFeeCode, claimValue)
-    return MoneyConverter.convertPenniesToPounds(fee.amount)
+    return this.calculateFee(issueFeeEvent, claimValue, onlineChannel)
```

Comment:
Please revert to async / await approach or remove async keyword from method declaration. We are supposed to use either async / await or native promise API.

Please do the same for `calculateHearingFee` function.